src/ethereum_test_fixtures/blockchain.py

```diff
@@ -401,6 +401,7 @@ class FixtureConfig(CamelModel):
     """Chain configuration for a fixture."""
 
     fork: str = Field(..., alias="network")
+    chain_id: int = Field(1, alias="chainid")
```

Comment:
I think this would be the sole `int` type in the entire fixture format.

Should we simply use `ZeroPaddedHexNumber` or `HexNumber` here to make it easier for parsers?

Reply:
I think that `ZeroPaddedHexNumber` would be the most consistent choice.